src/gtk/window.cpp

```diff
@@ -4240,6 +4243,14 @@ bool wxWindowGTK::GTKShowFromOnIdle()
     return false;
 }
 
+#ifdef __WINDOWS__
+WXHWND wxWindowGTK::GetHWND() const {
```

Comment:
Very minor, but wx style is to put braces on their own lines, i.e.
```suggestion
WXHWND wxWindowGTK::GetHWND() const
{
```

Reply:
Whoops, my bad... I didn't realize the difference in styling it seems.